### PR TITLE
ci: Backstop AOT tests

### PR DESCRIPTION
**Pull request by danobi**
This PR backstops AOT tests in CI. Not all the AOT test pass or get
skipped yet. Rather than wait for it to happen and then add in CI,
let's just add a subset in CI now to prevent any more backsliding.

We've already had quite a few regressions due to the codepaths
not being exercised in CI.

##### Checklist

- [ ] Language changes are updated in `man/adoc/bpftrace.adoc`
- [ ] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [ ] The new behaviour is covered by tests

### CODE REVIEW

File: .github/include/ci.py [hunk 1]
```diff
@@ -274,6 +276,34 @@ def test():
...
+                        "./tests/runtime-tests.sh",
+                        "--run-aot-tests",
+                        "--filter",
+                        "aot.*",
+                    ]
+                    + [
+                        "--skiplist_file",
+                        f"{root()}/{AOT_SKIPLIST_FILE}",
+                    ]
+                    if AOT_SKIPLIST_FILE
```
Comment (jordalgo) [inline .github/include/ci.py:295]: nit: why do we need this check here? Are there cases when it's ok if CI doesn't find this file even though we specify it.

File: tests/runtime/engine/runner.py [hunk 2]
```diff
@@ -93,6 +95,8 @@ def skip_reason(test, status):
             return "disabled by environment variable"
         elif status == Runner.SKIP_AOT_NOT_SUPPORTED:
             return "aot does not yet support this"
+        elif status == Runner.SKIP_IN_SKIPLIST:
+            return "disabled by entry in skiplist"
```
Comment (jordalgo) [inline tests/runtime/engine/runner.py:99]: super nit

Would also be cool to print the file name or path but maybe that's too verbose
```suggestion
            return "disabled by entry in skiplist file"
```

**Review by jordalgo — APPROVED** [review]
Minor question and nit but otherwise LGTM

**Comment by jordalgo** [comment]
Side note: In terms of test output, should we just show the number of tests skipped at the end instead of re-listing them all?

**Comment by danobi** [comment]
> Side note: In terms of test output, should we just show the number of tests skipped at the end instead of re-listing them all?

My personal preference would be to keep it as-is. I find the summary helpful - esp for when I build the skiplist which was just copy paste

**Pull request merged**